### DISCUSSION TITLE
Running RTSP Streaming Server on Xiao ESP32-S3 Sense

### DISCUSSION THREAD

**Discussion by jduanen** [pschatzmann/arduino-audio-tools#1532] [Q&A] [answered]
I'm having a bit of difficulty in getting the RTSP-related examples to compile. In particular, the examples/examples-communication/rtsp/communication-generator-rtsp/ example indicates that there's no definition of the PCMInfo class which is referenced in src/AudioLibs/RTSP.h.
I have failed to find any evidence of others having a problem, so it's probably pilot error on my part. Perhaps I'm missing some setup or additional libraries? I'm attempting to do the builds with the PlatformIO add-in to VSCode.
Any help would be welcomed.

**Comment by pschatzmann** [comment, marked as answer]
You can usually find the additional dependencies in the [class documentation](https://pschatzmann.github.io/arduino-audio-tools/classaudio__tools_1_1_r_t_s_p_output.html) and in [the sketch](https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-communication/rtsp/communication-generator-rtsp/communication-generator-rtsp.ino)!

**Reply by jduanen** [reply]
I had thought that I included the Micro-RTSP-Audio lib but found an error on my configuration.
Thanks for pointing me into the right direction. 
I really like your audio tools.
